single-function/lru-cache/submission.js

```diff
@@ -0,0 +1,65 @@
+class Node {
+  constructor (key, value) {
+    this.key = key;
```

Comment:
This looks kind of weird on the face of it, because it's meant to be a doubly-linked list node (if i'm reading this code right) but it doesn't mention `next` and `prev`.  Instead it seems to be organized around a hashtable-type lookup, but it's a `Node`...